components/slider/Slider.razor.cs

```diff
@@ -508,21 +591,33 @@ private async void OnMouseMove(JsonElement jsonElement)
 
         private async void OnMouseUp(JsonElement jsonElement)
         {
+            bool isMoveInEdgeBoundary = IsMoveInEdgeBoundary(jsonElement);
             if (_mouseDown)
             {
                 _mouseDown = false;
-                if (!IsMoveInEdgeBoundary(jsonElement))
+                if (!isMoveInEdgeBoundary)
                 {
                     await CalculateValueAsync(Vertical ? jsonElement.GetProperty("pageY").GetDouble() : jsonElement.GetProperty("pageX").GetDouble());
                     OnAfterChange?.Invoke(CurrentValue);
                 }
             }
-            if (_edgeClicked != null)
+            if (_toolTipRight != null)
             {
-                _edgeClicked = null;
-                _initialLeftValue = _leftValue;
-                _initialRightValue = _rightValue;
+                if (_tooltipRightVisible != TooltipVisible)
+                {
+                    _tooltipRightVisible = TooltipVisible;
+                    _toolTipRight.Visible = TooltipVisible;
+                }
+
+                if (_tooltipLeftVisible != TooltipVisible)
+                {
+                    _tooltipLeftVisible = TooltipVisible;
+                    _toolTipLeft.Visible = TooltipVisible;
+                }
```

Comment:
This part looks probably weird and confusing, but this was the only way I could find to deal with keeping the `Tooltip` visible while dragging slider's edge. Additional concern here was that clicking on an edge would keep the `Tooltip` visible. 
So `_tooltipRightVisible` and `_tooltipLeftVisible` are passed to `Overlay` as parameters (to `Visible` property). Any change to one of those parameters either forces the `Overlay` to show or to hide itself (in `OnParametersSetAsync()`). Here though, I was trying to deactivate force showing (during dragging) but I did not want the `Tooltip` to vanish. That is why there is also a call to `Overlay.Visible` that is omitting the `OnParametersSetAsync` and forced hiding.
I also tried using only call to `Overlay.Visible` but that get's reset on each refresh, so `Tooltip` was vanishing during dragging.

Reply:
It's fine.

Reply:
> The only "beef" is that I get a compliant from VS that Component parameter 'Visible' should not be set outside of its component.

You can replace it with a public method, `SetVisible(visible: bool)` etc.

Reply:
Will do it.